### DISCUSSION TITLE
"Provisioned dashboard cannot be deleted" but still disappears from the UI

### DISCUSSION THREAD

**Discussion by samjewell** [grafana/grafana#40089] [Dashboards, Panels & Visualizations]
Context: Personally, with the advent of Single Page Apps and Reactive apps, I've come to expect that what's on the screen always reflects what's persisted on the backend, and that I should never need to refresh the page to see the correct version of what's persisted.

So with that background, the issue is as follows: (I've illustrated with a Grafana Cloud example here, but the same would be true for any provisioned dashboards I assume).
- Start a new Stack on Grafana Cloud
- Go to /dashboards to manage your dashboards, then select the "Grafana Cloud Billing/Usage" dashboard and delete it.

Observed:
- Warning notification "provisioned dashboard cannot be deleted"
- Page re-renders in the browser, and the **dashboard has now been removed from the list**
- If I hard-refresh the page, the dashboard reappears.

Expected:
- dashboard never disappears from the list shown on the page

There's a chance you'd like this reported as a bug instead - if you want me to move it just let me know and I can move. But my assumption is that you consider this kind of SPA reactivity more like feature work at this stage (based on the frequency that I do a hard-refresh in Grafana already).

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Reply by gelicia** [reply]
@samjewell If this is still something you can reproduce, let me know and I will make an issue for it

**Comment by samjewell** [comment]
I can't reproduce this- to me this is fixed, thanks @gelicia 🙏
